tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -20,7 +20,7 @@ class Camera(Base):
     _secure_camera_frame_locator = (By.CSS_SELECTOR, ".secureAppWindow.active[data-manifest-name='Camera'] iframe")
 
     # Controls View
-    _controls_locator = (By.CSS_SELECTOR, '.controls')
+    _controls_locator = (By.CSS_SELECTOR, '.controls.enabled')
```

Comment:
This change is not required. I tried several ways to fix this issue, so in one of them I probably changed this locators. I will undo this change.